src/app/shared/formly/types/password-field/password-field.component.ts

```diff
@@ -18,8 +18,10 @@ export class PasswordFieldComponent extends FieldType {
 
     field.validation = field.validation ?? {};
     field.validation.messages = {
+      password: 'form.password.password.invalid',
```

Comment:
this should be form.password.error.invalid
![image](https://user-images.githubusercontent.com/34165455/127128289-75d0e28e-9fd6-4734-91be-5386fe882eb6.png)